src/components/learning-guide.cjsx

```diff
@@ -146,26 +217,14 @@ LearningGuide = React.createClass
               Weaker
             </div>
             <div>{weaker}</div>
-            <Router.Link
-            to='viewPractice'
-            params={linkParams}
-            query={page_ids: worstPages}
-            className='btn btn-default metric-button'>
-              Practice Weaker
-            </Router.Link>
+            {if @state.student then weakerPracticeButton}
```

Comment:
I think stronger/weaker might be useful for the teacher to know. Can you remove the condition? Thoughts @Fredasaurus @jason-holmes ?